### ISSUE TITLE
New installer with Python 3.9 uninstalls Python 2.7 and 3.7

### ISSUE THREAD

**Issue by chrisvire** [sillsdev/aeneas-installer#57]
I checked which versions of Python I had on my computer before running the macOS installer.  I had Python 2.7 (for some legacy projects) and 3.7.  After the installer, I only had Python 3.9.  Are you intentionally uninstalling previous Python versions?

**Comment by danielbair** [comment]
No. That's unexpected. I never tested such a setup before. I am using the python installer directly from python.org though and maybe it looks for old python installations and uninstalls them (especially since python 2.7 reached end of life a few years ago, but I couldn't guess why python 3.7 was uninstalled).

**Comment by danielbair** [comment]
I will look into it.
